spec/index.md

```diff
@@ -504,13 +498,25 @@ Whilst strictly not an entity, the concept of an "account" from which money is s
 
 ### Location
 
-There is a frequent desire to label items with location, usually by attaching geographic codes for a region or area. This allows the spending or revenue to  be analysed by region or area. This geographic information can be introduced directly by classifying the item with a code, or, more frequently indirectly by associating a geographic code to e.g. an entity. For example, by labelling a supplier with their location one can then associate a spend with that supplier as spending in that location.
+There is a frequent desire to label items with location, usually by attaching geographic codes for a region or area. This allows the spending or revenue to be analysed by region or area. This geographic information can be introduced directly by classifying the item with a code, or, more frequently indirectly by associating a geographic code to e.g. an entity. For example, by labelling a supplier with their location one can then associate a spend with that supplier as spending in that location.
```

Comment:
This is a reference to a mechanism that I am aiming to remove.

https://github.com/openspending/fiscal-data-package/pull/153/files